src/schemas/21-526EZ-allclaims/schema.js

```diff
@@ -597,6 +666,72 @@ const schema = {
           }
         }
       }
+    },
+    form4142: {
+      type: 'object',
+      properties: {
+        limitedConsent: {
+          type: 'string'
+        },
+        providerFacility: {
+          type: 'array',
+          minItems: 1,
+          maxItems: 100,
+          items: {
+            type: 'object',
+            required: [
+              'providerFacilityName',
+              'treatmentDateRange',
+              'providerFacilityAddress'
+            ],
+            properties: {
+              providerFacilityName: {
+                type: 'string',
+                minLength: 1,
+                maxLength: 100
+              },
+              treatmentDateRange: {
+                $ref: '#/definitions/dateRangeAllRequired'
+              },
+              providerFacilityAddress: {
+                type: 'object',
+                required: ['street', 'city', 'country', 'state', 'postalCode'],
+                properties: {
+                  street: {
+                    type: 'string',
+                    minLength: 1,
+                    maxLength: 20
+                  },
+                  street2: {
+                    type: 'string',
+                    minLength: 1,
+                    maxLength: 20
+                  },
+                  city: {
+                    type: 'string',
+                    minLength: 1,
+                    maxLength: 30
+                  },
+                  postalCode: {
+                    type: 'string',
+                    pattern: '^\\d{5}(?:([-\\s]?)\\d{4})?$'
+                  },
+                  country: {
+                    type: 'string',
+                    enum: baseAddressDef.properties.country.enum,
```

Comment:
Could we define `country` as `baseAddressDef.properties.country`?

Reply:
Yeah, that'd make sense, wouldn't it? States as well.